tests/components/stream/test_worker.py

```diff
@@ -87,13 +92,18 @@ def __next__(self):
             raise StopIteration
         self.packet += 1
 
-        class FakePacket:
+        class FakePacket(bytearray):
+            # Be a bytearray so that memoryview works
+            def __init__(self):
+                super().__init__(2)
```

Comment:
2 here and 3 below?

Reply:
We only need 2 bytes for the memoryview to work, but test checks that `size` is >2 (this is the same check in ffmpeg), so I just set it to 3.

Reply:
Seem they are related to same thing. So should be equal.

Reply:
I agree it's odd, but ffmpeg uses `> 2` for its test. Are you saying we should just use `>= 2` and diverge from ffmpeg?

Reply:
No I'm saying you should allocate 3 as well.

Reply:
Why? We're not accessing any other part of the buffer.

Reply:
Because your fake structures say it is this long. Your fake structure doesn't match a possible state of pyav.

Reply:
Sure, I'll change it just to keep it consistent. But this is just a hack to make the test work anyway - the packet data that's accessible here is not really valid.